### ISSUE TITLE
Rolling Update fails with exception on NFS task because of a bug in the module

### ISSUE THREAD

**Issue by styleart** [ceph/ceph-ansible#4045]
**Bug Report**

What happened:

Rolling update fails while executing the task `TASK [ceph-nfs : generate ganesha configuration file]` with an exception (`-vvv` output): 

```
MissingSectionHeaderError: File contains no section headers.
file: <???>, line: 4
u'NFS_Core_Param\n'

fatal: [ceph1]: FAILED! => 
  msg: Unexpected failure during module execution.
  stdout: ''
```

How to reproduce it (minimal and precise):
Run the `rolling-update.yaml` playbook with configured NFS in the running cluster.
Original mimic installation with branch `stable-3.2` on Ubuntu Bionic.
Rolling update with branch `stable-4.0` auf Nautilus on Ubuntu Bionic.

Since there are still no NFS packages for bionic available, I used the one suggested in comment https://github.com/ceph/ceph-ansible/issues/3789#issuecomment-487315017. 

NFS configuration file `group_vars/nfss.yaml`:

```
ceph_nfs_enable_service: true
nfs_obj_gw: true
nfs_file_gw: false 
ceph_nfs_dynamic_exports: true
ceph_nfs_log_file: "/var/log/ganesha/ganesha.log"
ceph_nfs_rados_backend: true
ceph_nfs_rados_export_index: "ganesha-export-index"
ceph_nfs_rgw_export_id: 20134
ceph_nfs_rgw_pseudo_path: "/cephobject"
ceph_nfs_rgw_protocols: "3,4"
ceph_nfs_rgw_access_type: "RW"
ceph_nfs_rgw_user: "cephnfs"
ceph_nfs_rgw_squash: "Root_Squash"
rgw_client_name: client.rgw.{{ ansible_hostname }}
```

**Environment**:
* OS: Ubuntu 18.04.2 LTS
* Kernel: Linux ceph1 4.15.0-50-generic #54-Ubuntu
* Ansible version: 2.8.0
* ceph-ansible version: stable-4.0
* Ceph version: before update: 13.2.5, after update: 14.2.1 (broken upgrade state)

**Comment by styleart** [comment]
@dsavineau thanks for the fast fix! will it be also merged into `stable-4.0` branch?

**Comment by dsavineau** [comment]
@styleart yes it has be backported into stable-4.0 as well https://github.com/ceph/ceph-ansible/pull/4047
